packages/one-app-bundler/bin/postProcessOneAppBundle.js

```diff
@@ -39,8 +39,11 @@ module.exports = async function postProcessBuild() {
   };
   const { hash } = await hashElement(tmpPath, options);
   const buildVersion = `${version}-${hash.slice(0, 8)}`;
-  const modernBrowserChunkAssets = require(path.resolve(pkgPath, '../.webpack-stats.browser.json')).assetsByChunkName;
-  const legacyBrowserChunkAssets = require(path.resolve(pkgPath, '../.webpack-stats.legacyBrowser.json')).assetsByChunkName;
+  if (!fs.existsSync(path.resolve(pkgPath, '../.build-stats'))) {
+    fs.mkdirSync(path.resolve(pkgPath, '../.build-stats'));
+  }
+  const modernBrowserChunkAssets = require(path.resolve(pkgPath, '../.build-stats/.webpack-stats.browser.json')).assetsByChunkName;
+  const legacyBrowserChunkAssets = require(path.resolve(pkgPath, '../.build-stats/.webpack-stats.legacyBrowser.json')).assetsByChunkName;
```

Comment:
solution could be simplified and make it a bit more readable

```suggestion
  const buildStatsPath = path.resolve(pkgPath, '../.build-stats');

  if (!fs.existsSync(buildStatsPath)) {
    fs.mkdirSync(buildStatsPath);
  }
  const modernBrowserChunkAssets = require(path.resolve(buildStatsPath, '.webpack-stats.browser.json')).assetsByChunkName;
  const legacyBrowserChunkAssets = require(path.resolve(buildStatsPath, '.webpack-stats.legacyBrowser.json')).assetsByChunkName;
```